sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FieldValue.java

```diff
@@ -70,6 +72,9 @@ public FieldValue(final Object value, final FieldValueType valueType) {
             case MAP:
                 formFieldMap = (Map<String, FormField>) value;
                 break;
+            case SELECTION_MARK:
+                selectionMark = (FormSelectionMark) value;
```

Comment:
I'm sure I am missing something. but how is this cast working? `value` should only return the state of the selection mark (an enum), but here we are casting it to the whole type. What are the values for text, bounding box, etc?

Reply:
Yes. It should be SELECTION_MARK_STATE instead of selection mark.